### PR TITLE
:sparkles: WoolyPooly Presence

### PR DESCRIPTION
**Pull request by TheGamerzs**

### CODE REVIEW

**Comment by TheGamerzs** [comment]
image

**Comment by TheGamerzs** [comment]
image

File: websites/W/WoolyPooly/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  "description": {
+    "en": "WoolyPooly is one of the most profitable pools with our focus on minimal commission fees for PPLNS & SOLO reward systems."
+  },
+  "url":
+    "woolypooly.com",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/X2sf9rc.png",
+  "thumbnail": "https://i.imgur.com/RVtEVMf.png",
+  "color": "#3d89f5",
+  "tags": ["other"],
```
Comment (Bas950) [inline websites/W/WoolyPooly/dist/metadata.json:16]: pretty sure you can come up with some better tags...

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/W/WoolyPooly/dist/metadata.json (hunk 1 above)
Comment (TheGamerzs) [reply websites/W/WoolyPooly/dist/metadata.json:16]: I thought the tags were only ones on the store

Comment (TheGamerzs) [reply websites/W/WoolyPooly/dist/metadata.json:16]: nvm, I'm thinking of category

**Review by Bas950 — APPROVED** [review]

**Review by ririxidev — APPROVED** [review]

**Pull request merged**